index.bs

```diff
@@ -2390,7 +2387,7 @@ defined as follows:
               <code>|desc|.{{USBPermissionDescriptor/filters}}</code>, continue
               to the next |device|.
            2. If <code>|desc|.{{USBPermissionDescriptor/exclusionFilters}}</code>
-              is set and |device| <a>matches a device filter</a> in
+              is not empty and |device| <a>matches a device filter</a> in
               <code>|desc|.{{USBPermissionDescriptor/exclusionFilters}}</code>,
```

Comment:
Same here. No need to check whether the list is empty. A device doesn't **match a filter** in a list if the list doesn't have any entries.

Reply:
Fixed.